src/styles/aui/_aui.navigation-menu.scss

```diff
@@ -0,0 +1,427 @@
+              }
+            }
+          }
+        }
+
+        .o-sub-menu__content-wrapper {
+          width: $menu-undocked-width;
+        }
+      }
+    }
+  }
+}
+
+.o-menu .o-menu__link,
+.o-menu .m-nav-list a {
+  white-space: nowrap;
+
+  &,
+  &:active,
+  &:link,
+  &:hover {
+    color: $text-color-light;
+    text-decoration: none;
+  }
+}
+
+.o-menu > .o-menu__tabs-wrapper {
+  align-content: space-between;
+  align-items: center;
+  background: $white;
+  border-top: 1px solid $border-color;
+  display: flex;
+  position: relative;
+  width: 100%;
+  z-index: layer('header');
+
+  // Only display the first 2 items on a mobile screen
+  // and ignore the last element, since it will show a 'more...' menu tab
+  & > .o-menu-tab:nth-child(n + 3) {
+    &:not(:last-child) {
+      display: none; //We will need to  display the 'more...' tab
+
+      @include at($screen-sm) {
+        display: block; // Just display all on desktop since we will have enough space
+      }
+    }
+
+    &:last-child {
+      display: block;
+
+      @include at($screen-sm) {
+        display: none; // Don't display the 'more' tab on desktop
+      }
+    }
+  }
+
+  @include at($screen-sm) {
+    flex-direction: column;
+    height: auto;
+    width: auto;
+    z-index: layer('pane-overlay');
+  }
+}
+
+.o-menu .o-menu-tab {
+  border-right: 1px solid $border-color;
+  cursor: pointer;
+  flex-basis: 33%;
+  flex-grow: 1;
+  height: 100%;
+  position: relative;
+  width: 33%;
+
+  &:last-child {
+    border-right: none;
+  }
+
+  &.active {
+    .o-menu-tab__wrapper {
+      overflow-y: auto;
+    }
+
+    .a-tab-submenu-icon {
+      transform: rotate(180deg);
+    }
+
+    & > .o-menu-tab__wrapper > .o-menu-submenu > .o-sub-menu__content-wrapper:first-of-type {
+      display: block;
+      max-height: $submenu-height;
+      overflow: visible;
+    }
+  }
+
+  @include at($screen-sm) {
+    border-bottom: 1px solid $border-color;
+    border-right: none;
+    width: 100%;
+  }
+
+  & > .o-menu-tab__wrapper {
+    overflow: hidden;
+
+    &:focus-within {
+      @include a-button-outline;
+    }
+
+    & > .o-menu-tab__header {
+      //  @include a-button;
+      @include a-button-outline($btn-primary-outline-color, $btn-primary-outline-border, true);
+      align-items: center;
+      color: $text-color-light;
+      display: flex;
+      flex-direction: column;
+      font-weight: bold;
+      height: $mobile-menu-height;
+      justify-content: center;
+      text-decoration: none;
+
+      // Support for a margin for iPhone X
+      @supports (margin-bottom: env(safe-area-inset-bottom)) {
+        margin-bottom: env(safe-area-inset-bottom);
+      }
+
+      &:hover {
+        background-color: $background-color-light;
+        color: $text-color-light;
+      }
+
+      .o-menu-tab__title {
+        align-items: center;
+        display: flex;
+        flex-direction: column;
+        text-align: center;
+      }
+
+      .a-icon {
```

Comment:
Zelfde opmerking als hierboven: `a-icon` bestaat niet, dus ofwel via inheritance ofwel iets in de trend van `.o-menu__icon`.